python/ngen_cal/src/ngen/cal/calibratable.py

```diff
@@ -80,10 +83,11 @@ def check_point_file(self) -> 'Path':
         """
         return Path('{}_parameter_df_state.parquet'.format(self.id))
 
-    def check_point(self, path: 'Path') -> None:
+    def check_point(self, info: JobMeta, iteration: int) -> None:
```

Comment:
Might wanna switch this around too

Reply:
now you really are getting picky 🤣

Reply:
changed in 50b3f05d93dd96d8761a421e060f02f07b86bc00